### DISCUSSION TITLE
what is errorcode of canellation in quick login?

### DISCUSSION THREAD

**Discussion by soengames** [lupidan/apple-signin-unity#161] [Q&A]
hello.

I would like to know the error code when cancel on the screen displayed during quick login.

I'm currently getting 1001, but this doesn't seem right.

Where can I find out?

**Comment by lupidan** [comment]
In the code

https://github.com/lupidan/apple-signin-unity/blob/e39981bca07f08f068231f651e1e8d0f69397222/AppleAuth/Enums/AuthorizationErrorCode.cs#L16
